nucypher/utilities/prometheus/collector.py

```diff
@@ -158,20 +159,20 @@ def __init__(
         self.contract_registry = contract_registry
         self.eth_endpoint = eth_endpoint
 
-    def initialize(self, metrics_prefix: str, registry: CollectorRegistry) -> None:
+    def initialize(self, registry: CollectorRegistry) -> None:
         self.metrics = {
             "active_stake_gauge": Gauge(
-                f"{metrics_prefix}_associated_active_stake",
+                "associated_active_stake",
                 "Total amount of T staked (adapted NU/KEEP and liquid T)",
```

Comment:
Now that legacy stakes are in the process of being deprecated.
```suggestion
                "Total amount of T staked",
```